### ISSUE TITLE
Should we set eslint root: true for ckeditor5 repos?

### ISSUE THREAD

**Issue by oskarwrobel** [ckeditor/ckeditor5-dev#259]
`"root": true`
> By default, ESLint will look for configuration files in all parent folders up to the root directory. This can be useful if you want all of your projects to follow a certain convention, but can sometimes lead to unexpected results. To limit ESLint to a specific project, place "root": true inside the eslintConfig field of the package.json file or in the .eslintrc.* file at your project’s root level. ESLint will stop looking in parent folders once it finds a configuration with "root": true.

**Comment by Reinmar** [comment]
As I understand the problem is that the configs which are defined in ckeditor5 repos may be extended by ESLint with configs which are defined in some parent project in which CKEditor packages are cloned?

**Comment by ma2ciek** [comment]
Yes, `ckeditor5/.eslintrc.js` config is merged with `ckeditor5/pacakges/@ckeditor/ckeditor5-*/.eslintrc` when running eslint inside the `ckeditor5/pacakges/@ckeditor/ckeditor5-*/`

**Comment by Reinmar** [comment]
Hm... that's actually useful :D But ok, I guess we can enable that option.
